codewars/8kyu/arrayFilter.js

```diff
@@ -0,0 +1,31 @@
+let numbers = [1, 3, 5, 1, 8, 9, 11];
+let usersAge = [12, 23, 18, 13, 45];
+let names = ['Matt', 'Beata', 'Ben', 'Tom', 'Zuza'];
+
+function checkNameLength(name){
+    return name.length > 3;
+}
```

Comment:
try to leave a space line above and below the function for better code readability

Reply:
Done